src/item/FireworkStar.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item;
+
+use pocketmine\block\utils\DyeColor;
+use pocketmine\color\Color;
+use pocketmine\data\SavedDataLoadingException;
+use pocketmine\nbt\tag\CompoundTag;
+use pocketmine\utils\Binary;
+
+class FireworkStar extends Item{
+
+	protected const TAG_EXPLOSION = "FireworksItem"; //TAG_Compound
+	protected const TAG_CUSTOM_COLOR = "customColor"; //TAG_Int
```

Comment:
Is it really a `c` at the start and not a `C` ?

Reply:
yep